lib/utilities/lint-fix.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+const chalk = require('chalk');
+const execa = require('execa');
+
+async function run() {
+  let lintFixScriptName = `lint:fix`;
+  let cwd = process.cwd();
+  let npmRunResult = await execa('npm', ['run'], { cwd });
+  let lintFixScriptRegex = new RegExp(`^  ${lintFixScriptName}$`, 'm');
+  let hasLintFixScript = lintFixScriptRegex.test(npmRunResult.stdout);
```

Comment:
Hmm, why do we need to run `npm run` just to figure out if they have a lint:fix script? Couldn't we directly read the JSON file to see?